medcat/utils/preprocess_snomed.py

```diff
@@ -249,7 +249,8 @@ def relationship2json(self, relationshipcode, output_jsonfile):
                     relationship[_].append(v['sourceId'])
                 else:
                     pass
-            output_dict.update(relationship)
+                    output_dict = {key: output_dict.get(key, []) + relationship.get(key, []) for key in
```

Comment:
I don't know if it's just the online diff view but the new expression indentation makes it go into the `else` clause after the `pass` statement, which would (somewhat obviously) mean this never gets run.

Reply:
Good spot thanks!